tsconfig.json

```diff
@@ -1,11 +1,12 @@
 {
   "compileOnSave": false,
   "compilerOptions": {
+    "outDir": "out-tsc/lib",
     "baseUrl": "./",
     "paths": {
       "@ngxs/*": ["packages/*"]
     },
-    "noUnusedLocals": true,
+    "strict": true,
```

Comment:
It's breaking changes?

Reply:
Does this count as breaking changes? It is just a change to the compiler options.
I don't think it will affect the users of this lib... unless I'm mistaken?

Reply:
it won't affect users of the lib. just anyone working ON the lib

Reply:
at least I think I am right lol